client/main/tableview.js

```diff
@@ -726,7 +761,7 @@ const TableView = SilkyView.extend({
             columnType = columnType[0].toUpperCase() + columnType.substring(1);
             let err = {
                 title: 'Column is not editable',
-                message: columnType + ' columns may not be edited',
+                message: columnType + ' columns may not be edited.\nPress "F3" to modify column properties.',
```

Comment:
maybe ditch this, let's discuss it, and we'll put it in a separate commit. (generally, we include additional information if it helps the person overcome an issue, but pressing F3 won't help them edit that cell. so we may want to rethink this ... though i appreciate where you're coming from).

Reply:
This was really only for the calculated variable in which case it makes more sense but i appreciate it is a response for all editing issues.

Reply:
yeah, but if they're trying to edit that value, they're misunderstanding what computed columns are all about ... and bringing up the variable editor will likely add to their confusion (some people will be able to figure it out from the variable editor, but i think most not). if anything i'd rather direct them to some help/documentation.

Reply:
i'm always surprised how *few* people [want to] use F-keys too.